src/v/kafka/server/handlers/produce.cc

```diff
@@ -186,6 +186,8 @@ static partition_produce_stages partition_append(
                     p.error_code = error_code::none;
                     partition->probe().add_records_produced(num_records);
                     partition->probe().add_bytes_produced(num_bytes);
+                    partition->probe_v2().add_produce_bytes_cluster_lvl(
```

Comment:
The requirements for this new metric are a combination of the old metrics generated from `partition_probe::add_bytes_produced()` and `partition_probe::add_bytes_fetched()`. But those metrics were at the cluster level. I still need to figure out where to put the topic level metrics.